src/Windows.Core/LIST_ENTRY.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    [StructLayout(LayoutKind.Sequential)]
+    public unsafe struct LIST_ENTRY
+    {
+        public LIST_ENTRY* Flink;
```

Comment:
I tried being clever with [MarshalAs(LPStruct)] on a different struct, but it ended up not working.